db/migrate/20130507212348_create_records.rb

```diff
@@ -1,16 +1,7 @@
-# The majority of The Supplejack Website code is Crown copyright (C) 2014, New Zealand Government,
-# and is licensed under the GNU General Public License, version 3. Some components are 
-# third party components that are licensed under the MIT license or other terms. 
-# See https://github.com/DigitalNZ/supplejack_website for details. 
-# 
-# Supplejack was created by DigitalNZ at the National Library of NZ and the Department of Internal Affairs. 
-# http://digitalnz.org/supplejack
+# frozen_string_literal: true
 
 class CreateRecords < ActiveRecord::Migration
   def change
-    create_table :records do |t|
-
-      t.timestamps
-    end
+    create_table :records, &:timestamps
```

Comment:
Holy crap black magic, block-to-proc'ing a migration! (totally ok with this just never occurred to me)

Reply:
rubocop did this by itself ^^